scripts/factory-hooks/post-settings-php/memcache.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * Example implementation of ACSF post-settings-php hook.
+ *
+ * @see https://docs.acquia.com/site-factory/tiers/paas/workflow/hooks
+ */
+
+// Use ACSF internal settings site flag to apply memcache settings.
+if (getenv('AH_SITE_ENVIRONMENT') && 
+	isset($site_settings['flags']['memcache_enabled']) &&
+	isset($settings['memcache']['servers'])
+) {
+  require DRUPAL_ROOT . '/../vendor/acquia/blt/settings/memcache.settings.php';
```

Comment:
I believe this issue is addressed directly in #3006. BLT settings are included in a pre-settings-php hook intentionally to set the request variable values used at various stages of the bootstrap, install, and update process.

Reply:
the documentation was broken out into separate tickets because the changes were so significant. Over the last few months it has been code and content reviewed by hosting and other relevant stakeholders, and published after approvals in https://backlog.acquia.com/browse/PUB-17066. These updates are live now and split across multiple pages at https://docs.acquia.com/acquia-cloud/performance/memcached/enable/ , https://docs.acquia.com/acquia-cloud/performance/memcached/pecl/ , https://docs.acquia.com/acquia-cloud/performance/memcached/drupal/ etc. 

The ACSF specific docs are in https://backlog.acquia.com/browse/PUB-17068 with additional internal and external information about  `memcache_enabled`, however it is an internal flag used in hosting and ACSF in the Acquia database config so we should only link to that page once it is published. 

Both docs sections include this exact settings configuration because it is also becoming the baseline configuration for all Acquia customers very soon and has gone through many months of production testing on ACSF and ACE on BLT and non-BLT projects so any changes to actual config, hook order, etc need to go through hosting and CL review.

Reply:
circling back to this because we made a lot of progress this weekend on the beta release which required testing this config on ACSF and the team was unable to reproduce this issue - @danepowell can you clarify how you tested and deployed this PR to ACSF including the environment (prod / 01live vs. a lower environment and if you followed the steps about enabling the flag from internal docs?

Reply:
Just to close the loop on documentation... sorry if I wasn't clear before, but the reason we need documentation is because we need to know how to enable the "site flag" for memcache in order to test this. I finally found a Confluence page that contains more details, anyone who needs a link can ping me internally.